Bottom: c82ca703f6a3fe8d475692b340fabcde67ab4b43
Top:    db58dde7259196e78fca4e373069d21b50b6653b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 17:27:04 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index ed10b50..79a914a 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
